core/trino-main/src/main/java/io/trino/eventlistener/EventListenerModule.java

```diff
@@ -27,5 +28,7 @@ public void configure(Binder binder)
     {
         configBinder(binder).bindConfig(EventListenerConfig.class);
         binder.bind(EventListenerManager.class).in(Scopes.SINGLETON);
+        newExporter(binder).export(EventListenerManager.class)
+                .as(generator -> generator.generatedNameOf(EventListenerManager.class, "eventListenerManager"));
```

Comment:
Use `org.weakref.jmx.ObjectNameGenerator#generatedNameOf(java.lang.Class<?>)`. No need to pass this extra string.